wayland/install-wayland.txt

```diff
@@ -0,0 +1,72 @@
+NOTE: This article covers only Wayland - the protocol, which is still
+      experimental.
+      Bugs and quirks expected!
+
+
+Preamble
+________________________________________________________________________________
+
+Though KISS does not support Wayland officially it is no problem to make it
+capable through personal or 3rd party repositories [0].
+Some packages pickup Wayland automatically, others need to set the right confi-
+gure flags to enable it.
+Gtk+3 and Qt5 are known to allready offer good Wayland support and most of KISS' 
+browser stack is working well.
+
+
+Packages which need modifications
+________________________________________________________________________________
+
+<gtk+3>   --enable-wayland-backend
+
+<firefox> --enable-default-toolkit=cairo-gtk3-wayland
+
+<mesa>    -Dplatforms=x11,drm,wayland
+
+
+Packages which pickup Wayland automatically at buildtime
+[needs expansion and verification]
+________________________________________________________________________________
+
+<cairo> <intel-vaapi-driver> <libva> <mpv> <pango> <sdl2> <webkit2gtk>
+
+
+Installation
+________________________________________________________________________________
+
+NOTE: From this point it is expected that <wayland> and <wayland-protocols> are
```

Comment:
We shouldn't use `<` and `>` directly as HTML is still interpreted. You can use `&lt;` and `&gt;` in their place if they're needed.